nsv13/code/modules/overmap/weapons/projectiles_fx.dm

```diff
@@ -168,7 +168,7 @@ Misc projectile types, effects, think of this as the special FX file.
 	name = "triton cruise missile"
 	icon_state = "conventional_missile"
 	speed = 1
-	damage = 175
+	damage = 250
```

Comment:
This makes them deal an equal amount of damage as regular torpedoes, a bit high in my opinion. Maybe consider lowering the damage of PDC rounds or increasing the missile's health slightly instead of doing this?

Reply:
Problem with raising the health is that they'll be even more annoying for the main ship to shoot down. Pretty much requiring the missiles to be going perfectly straight and for PDC's to already be firing in it's direction before they enter visual range.
A speed increase would be the same issue.
Now that you mention it a damage increase isn't the ideal solution either.
More missiles could be a good balance? Having 6 missiles would deal less than the proposed change but still close enough. Only problem I can see there however is getting the missiles in the first place from muni. Since the whole idea of this would be circling around, getting ideal launch parameters and then RTBing once you're out. However for RTBing to be at all useful you need a steady supply of missiles which isn't very easy to acquire. Would have to see how the players adapt, if they don't the change wouldn't be very feasible.

Reply:
If you like the idea of raising the default missile count I'd also have to change the upgraded launcher's count as it'd be a downgrade at that point.